.github/workflows/test.yml

```diff
@@ -152,9 +152,7 @@ jobs:
         with:
           PATTERNS: |
             examples/**
-            cli/**/*.go
-            cli/go.mod
-            cli/go.sum
```

Comment:
I removed trigger on `cli/**`, because the script uses the `turbo` installed in the example, rather than building locally. So PRs that change how turbo works isn't really getting tested against examples. We may still want to do that in a daily cron job though. This setup will make it easier to do so.

Reply:
Makes sense